Sources/Kitura/Router.swift

```diff
@@ -88,9 +88,14 @@ public class Router {
     /// ```
     /// - Parameter mergeParameters: Optional parameter to specify if the router should be able to access parameters
     ///                                 from its parent router. Defaults to `false` if not specified.
-    public init(mergeParameters: Bool = false) {
+    /// - Parameter useResourceServer: Optional parameter to start and use the
+    ///                                FileResourceServer to serve the default
+    ///                                "Welcome to Kitura" page and related
+    ///                                assets.
+    public init(mergeParameters: Bool = false, useResourceServer: Bool = false) {
```

Comment:
Please can we name this param `enableWelcomePage`?  I think it is more descriptive (also the 'ResourceServer' is an internal implementation detail).

Also, having considered it more, the default should be `true`, to make this an opt-out rather than an opt-in.  We want to maintain the existing behaviour by default, otherwise this would have to wait until Kitura 3.  (we could consider turning it off at that time).